.upstream-tests/test/std/containers/views/mdspan/mdspan.extents.cons/array.pass.cpp

```diff
@@ -49,18 +49,18 @@ int main(int, char**)
     test_array_con< cuda::std::tuple_element_t< 4, extents_test_types > >();
     test_array_con< cuda::std::tuple_element_t< 5, extents_test_types > >();
 
-    static_assert( is_array_cons_avail_v< cuda::std::dextents<int,2>, int   , 2 > == true  );
+    static_assert( is_array_cons_avail_v< cuda::std::dextents<int,2>, int   , 2 > == true , "" );
```

Comment:
Just FYI, these are changes to support C++14 -- C++17 implements one-argument `static_assert`.